packages/migrate/src/commands/MigrateDiff.ts

```diff
@@ -42,6 +43,11 @@ ${italic('From and To inputs (1 `--from-...` and 1 `--to-...` must be provided):
   --from-migrations        Path to the Prisma Migrate migrations directory
   --to-migrations
 
+  --from-local-d1          Automatically locate the local Cloudflare D1 database, see ${link(
+    'https://pris.ly/d/diff-cloudflare-d1',
```

Comment:
This does no exists yet?

Reply:
Thanks, removed it in https://github.com/prisma/prisma/pull/23662/commits/a44545f3821c00e1c19f4a8355030217f1874f03.